tests/test_custom_policy.py

```diff
@@ -12,7 +12,7 @@
 class CustomCommonPolicy(FeedForwardPolicy):
     def __init__(self, *args, **kwargs):
         super(CustomCommonPolicy, self).__init__(*args, **kwargs,
-                                           layers=[8, 8],
+                                           net_arch=[dict(vf=[8, 8], pi=[8, 8])],
```

Comment:
```suggestion
                                           net_arch=[8, dict(vf=[8, 8], pi=[8, 8])],
```
That way you can test the shared option too ;)